listenbrainz/webserver/views/api.py

```diff
@@ -105,7 +105,7 @@ def submit_listen():
 
 
 @api_bp.route("/user/<user_name>/listens", methods=['GET', 'OPTIONS'])
-@crossdomain()
+@crossdomain(headers="Authorization")
```

Comment:
Once the crossdomain changes work, these headers can be removed from everywhere.